### PR TITLE
Add notebook for projecting a WRF dataset

### PR DESCRIPTION
**Pull request by kyleredilla**
This PR adds a `project_wrf.ipynb` jupyter notebook that demonstrates how to project a WRF dataset using the standard WRF geogrid file. It does not attempt to make a CF-compliant netcdf dataset, it simply shows how to generate the x/y coordinates of the data in the WRF CRS and assign them to a file that does not have them.

### CODE REVIEW

File: project_wrf.ipynb [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+    "# SERDP WRF geogrid file\n",
+    "geo_ds = xr.open_dataset(\"/atlas_scratch/kmredilla/WRF/serdp/geo_em.d01.nc\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a34eb3a2-8af7-49a4-88ce-2ebaad99535b",
+   "metadata": {},
+   "source": [
+    "Create the projection info fro reprojecting lat/lon of WRF grid centerpoint to WRF CRS:"
```
Comment (charparr) [inline project_wrf.ipynb]: fix typo fro to for?

File: project_wrf.ipynb [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+   "id": "fa35cf79-2912-43b3-98a7-23f257b58732",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create the WRF projection\n",
+    "wrf_proj = Proj(\n",
+    "    proj=\"lcc\", # projection type: Lambert Conformal Conic\n",
+    "    lat_1=geo_ds.TRUELAT1, lat_2=geo_ds.TRUELAT2, # Cone intersects with the sphere\n",
+    "    lat_0=geo_ds.MOAD_CEN_LAT, lon_0=geo_ds.STAND_LON, # Center point\n",
+    "    a=6370000, b=6370000 # This is it! The Earth is a perfect sphere\n",
```
Comment (charparr) [inline project_wrf.ipynb:61]: great use of comments!! seriously love it. Once you start intersecting cones and spheres...you are earning your geo-dollars that's for sure

File: project_wrf.ipynb [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    "plt.imshow(arr[0], interpolation=\"none\")\n",
+    "plt.colorbar()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2eb4ed98-0170-4bfe-adc9-07cfb69c2580",
+   "metadata": {},
+   "source": [
+    "Error looks good - white noise! \n",
```
Comment (charparr) [inline project_wrf.ipynb:267]: nerd in me wants to see the histogram - but that's probably overkill for this application.

**Review by charparr — COMMENTED** [review]
You credit that blog post but this notebook is the real deal blog post IMO. I learned quite a bit in a short amount of time. Take a look at the typo I commented on.

File: project_wrf.ipynb (hunk 1 above)
Comment (kyleredilla) [reply project_wrf.ipynb]: nice catch!

File: project_wrf.ipynb (hunk 2 above)
Comment (kyleredilla) [reply project_wrf.ipynb:61]: Hah honestly these comments are from the blog.. but super helpful here for sure!

**Pull request merged**